.scripts/seeder/seed.php

```diff
@@ -23,6 +23,11 @@
 	throw new LogicException("Seeds should not be run with a logged in user");
 }
 
+// disable mailer
 _elgg_services()->setValue('mailer', new \Zend\Mail\Transport\InMemory());
 
-_elgg_services()->seeder->seed();
\ No newline at end of file
+// set seeder directory
+elgg_set_config('seeder_local_image_folder', __DIR__ . '/images/');
```

Comment:
This just adds more global state. Inject an options array into constructor instead.

Reply:
will follow up in #12224